app/src/msw/handler.ts

```diff
@@ -0,0 +1,3 @@
+import { packetGroupHandlers } from "./handlers/packetGroupHandlers";
+
+export const handlers = [...packetGroupHandlers];
```

Comment:
it can be used as a service worker to mock api calls outisde of just testing as well, thus it is placed outside of tests...  Also purposely atomised as each handler would be for each endpoints requests

Reply:
Yeah we discussed that - could we maybe have a brief reference in the README, describing that this is available when mocking API is required in dev scenarios when API not yet available.